src/lib/services/digital-asset.service.ts

```diff
@@ -86,15 +87,28 @@ function initializeLSP7Proxy(
       const contract = await new LSP7DigitalAssetInit__factory(signer).attach(
         result.receipt.contractAddress
       );
+
+      const gasEstimate = await contract.estimateGas[`initialize(string,string,address,bool)`](
```

Comment:
is it possible to get this `initialize(string,string,address,bool)` dynamically from the lsp-sc lib? Let's say the contract changes and this signature changes, would be nice to have it updated here too?

Reply:
or `contract.estimateGas.initialize(...)` ?

Reply:
Good point. This is actually because for some reason these functions are generated with string names so we need to access them like this. Im not sure why exactly but should look into changing it if possible 

![Screenshot 2021-11-30 at 10 59 25](https://user-images.githubusercontent.com/54543428/144026194-8b3a1056-20e9-4ed0-b36d-62cd16197a21.png)
as